citi-onboarding/DADe-onepage
# Feature/create footer

## What I did

- Create footer view (includes the .jsx file and the .css file)
- Add footer exports in the needed index.jsx files
- Call footer view in the App.jsx file
- Add Montserrat Regular 400 from google fonts into the App.css file

| Mockup | Real |
|--------|------|
![Captura de Tela_select-area_20210615142938](https://user-images.githubusercontent.com/61971582/122098254-ffa35b00-cde6-11eb-9c55-f960f741440b.png) | ![Captura de Tela_select-area_20210615143007](https://user-images.githubusercontent.com/61971582/122098259-029e4b80-cde7-11eb-8fb3-5b4d130c2476.png)

( I recommend holding the above images and dropping them in a new tab from the browser, to get a better look at them)


## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/create-footer`
   4. `git pull origin feature/create-footer`
   5. `npm install`
   6. `cd client/`
   7. `npm start`

3. In your browser go to http://localhost:3000 (probably will automatically open after the `npm start` command)

4. Check if the component looks just like the mockup in different screen sizes (except in a very small mobile screen, it will need a media query when we start developing the mobile version).  Also, check if after expading screen width past 1440px, the footer items stay in place, instead of the distance between them increasing.

5. Read the changed files section on GitHub and see if everything is correct

6. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;

**Resolves:** footer issue
